src/test/java/Homework14.java

```diff
@@ -0,0 +1,43 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+public class Homework14
```

Comment:
**According to the Java style guide:** 

1. Class names should be nouns, in mixed cases with the first letter of each internal word capitalized. Interfaces names should also be capitalized just like class names.
2. Use whole words and must avoid acronyms and abbreviations.

For example

![image](https://user-images.githubusercontent.com/104761482/209645889-69fca521-9584-4b99-b188-88075a406d35.png)

![image](https://user-images.githubusercontent.com/104761482/209646001-fbe75bed-8f5f-4c77-a18f-138de035555d.png)

Reply:
Ok. I will adjust. seems like such a small fix. Thanks for your input.